pycbc/workflow/grb_utils.py

```diff
@@ -643,7 +626,7 @@ def make_pygrb_injs_tables(workflow, out_dir, bank_file, off_file, seg_files,
     # Handle input/output for onsource/offsource
     else:
         src_type = 'offsource-trigs'
-        if on_file is not None:
+        if on_file:
             src_type = 'onsource-trig'
             # Onsource input file (passed as File instance)
             onsource_file = on_file
```

Comment:
What is the point of defining this extra variable? You could just pass `on_file` to `add_input_opt()` no?

Reply:
Ah, you mean `onsource_file`.  Yes, that is pointless.

Reply:
Yes sorry I meant `onsource_file`. And I think I spotted a similar case a few lines above this one.

Reply:
`fm_file = inj_file`... there it is

Reply:
I found two of those.  I grep-ed `if ` and piped it to `grep _file` to look for more, but this is it.